themes/ddbasic/ddbasic.info

```diff
@@ -101,8 +102,6 @@
 
   scripts[] = scripts/ting-object/ting-object.js
 
-  scripts[] = scripts/view.js
-
```

Comment:
Why has this been removed? It does not seem to be related to the change at hand and I cannot tell from the commit comments.

Reply:
That certainly looks like a mistake. The lines have been added again. https://github.com/ding2/ding2/pull/1406/commits/814ec9aeeb042b80bd021d664fafbc3906035b22